packages/pages/src/components/link/link.test.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import { Link } from "./link";
+
+describe("Link", () => {
+  it("renders component when given href + children", () => {
+    render(<Link href="https://yext.com">Click Me</Link>);
+  });
+
+  it("renders component when given partial cta and children", () => {
+    render(<Link cta={{ link: "https://yext.com" }}>Click Me</Link>);
+  });
+
+  it("renders component when given full cta prop and no children", () => {
+    render(<Link cta={{ link: "https://yext.com", label: "Click Me" }} />);
+  });
+
+  it("renders component when given full cta prop and no children", () => {
+    render(<Link cta={{ link: "https://yext.com" }} />);
+  });
+});
+
+// TODO: Add tests
```

Comment:
Do you plan to add these in this PR?

Reply:
I think if we're going to delete the TODO we should add an issue to add these tests

Reply:
but after our discussion in #151 it seems like it's fine to leave the TODOs